### DISCUSSION TITLE
Disable sort

### DISCUSSION THREAD

**Discussion by cool-RR** [saulpw/visidata#1761] [Q&A] [answered]
Say that I set up sorting on a table and I want to disable it so I would get the default sorting. (Maybe Shift-U doesn't work for some reason.) How can I do it?

**Comment by saulpw** [comment, marked as answer]
Alas, after it's sorted, you can only `Shift+U` to undo or `Ctrl+R` to reload the table again.

Before you've sorted though, you could use `i` to make an increment column, and then re-sort on that column to return to the original sort order.

**Reply by cool-RR** [reply]
Feature request: #1762
